### ISSUE TITLE
RuntimeError: check_and_blame_insufficient_funds_phase_5: only 1 input per player is allowed!

### ISSUE THREAD

**Issue by ProfessorVenkman** [Electron-Cash/Electron-Cash#1219]
Crash Report
============

This crash report was reported through the automatic crash reporting system 🤖

Traceback
--------------

```Python traceback
  File "/Users/calin/tmp/ec/electron-cash/lib/util.py", line 719, in run_with_except_hook
    run_original(*args2, **kwargs2)
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/client.py", line 77, in wrapper
    func(self)
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/client.py", line 207, in run
    self.start_protocol()
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/client.py", line 77, in wrapper
    func(self)
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/client.py", line 182, in start_protocol
    self.protocol.start_protocol()
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/round.py", line 100, in start_protocol
    self.protocol_loop()
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/round.py", line 131, in protocol_loop
    self.process_inbox()
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/round.py", line 181, in process_inbox
    handler()
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/round.py", line 439, in process_verification_and_submission
    ret = self.check_and_blame_insufficient_funds_phase_5(self.transaction)
  File "/Users/calin/tmp/ec/electron-cash/plugins/shuffle/round.py", line 875, in check_and_blame_insufficient_funds_phase_5
    raise RuntimeError('check_and_blame_insufficient_funds_phase_5: only 1 input per player is allowed!')

RuntimeError: check_and_blame_insufficient_funds_phase_5: only 1 input per player is allowed!
```

Reporter
------------

This issue was reported by 2 user(s):

| Electron Cash Version  | Python Version | Operating System  | Wallet Type  | Locale |
|---|---|---|---|---|
| 4.0.0  | 3.5.6 (default, Sep 20 2018, 11:49:45)  [GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] | Darwin-17.7.0-x86_64-i386-64bit | standard,imported_privkey | en_US | 
| 4.0.0  | 3.7.2 (default, Jan 10 2019, 23:51:51)  [GCC 8.2.1 20181127] | Linux-5.0.3-arch1-1-ARCH-x86_64-with-arch | standard | en_US | 

Additional Information
------------------------

The reporting user(s) did not provide additional information.

**Comment by cculianu** [comment]
This indicates to me the phase 0 "exclusive players" check is broken.

Will fix. High priority.

**Comment by cculianu** [comment]
Fixed in latest commit 5ce612f4a5b100890f935cb9513df51a69b0b169
